reactor-core/src/main/java/reactor/core/publisher/SinkEmptyMulticast.java

```diff
@@ -1,28 +1,24 @@
 package reactor.core.publisher;
```

Comment:
the license header is missing (was also missing in `VoidProcessor.java`... oops!)